python/lsst/ip/diffim/getTemplate.py

```diff
@@ -644,31 +634,32 @@ def run(self, coaddExposures, bbox, wcs):
         maskedImageList = []
         weightList = []
 
-        for coaddExposure in coaddExposures:
-            coaddPatch = coaddExposure.get()
+        for coaddExposure, dataId in zip(coaddExposures, dataIds):
```

Comment:
calling it `coaddExposure` instead of `coaddPatch` adds 7 extra lines to your PR.